code/modules/projectiles/guns/energy/stun.dm

```diff
@@ -53,3 +53,12 @@
 /obj/item/gun/energy/disabler/cyborg/newshot()
 	..()
 	robocharge()
+
+/obj/item/gun/energy/disabler/cyborg/cyborg_recharge(coeff, emagged)
```

Comment:
TG has a generic process for recharging a shot for all energy guns; might be necessary to prevent some of the energy guns on a few borgs from breaking.

Reply:
Do you know the name of the proc they use?

Reply:
It's just in the base level of `respawn_consumable`

Reply:
Ah, I believe we have something similar:

![image](https://user-images.githubusercontent.com/42044220/111241433-1b6e4980-85cb-11eb-8b9f-913d9fdf526a.png)

Which is ultimately called from the `on_recharge()` just below on line 61.

Reply:
Hmm, not the same, but or gun code is ancient by now...